virttest/virsh.py

```diff
@@ -1,3 +1,14 @@
+# This program is free software; you can redistribute it and/or modify
```

Comment:
suggest not add this license information, it is old file contributed by many others.
If you add declared license information here, it may bring some unnecessary controversy.

Reply:
It should not be contained in this patch. Fixed.